doc/design/dist/submit-job.md

```diff
@@ -0,0 +1,137 @@
+
+# PaddlePaddle Client
+PaddlePaddle client is command line tool, you can use a PaddlePaddle client to start a local train and submit a distributed training job to kubernetes cluster.
+
+The relation of PaddlePaddle, kubernetes and docker:
+
+<img src="./submit-job.png" width="500">
+
+
+# Running your training locally
+Execute `paddle local train` to run your local train.
+```bash
+paddle local train
+  --pcakage-path=./demo
+  --module=demo.train
+  --input=<input_dir>
+  --output=<output_dir>
+  --image=<paddle_image>
+  --e=NUM_PASS=4
+```
+- package-path: your trainer code python package
+- module: include a main function, trainer entrance.
+- input: input directory, for local train, it's a host path.
+- output: output directory, for local train, it's a host path.
+- image: paddlepaddle production image
+- e: environment varible
+
+When you start the local train, the client starts a docker container like:
+```bash
+  docker run --rm
+  -v <input_dir>:/train/input
+  -v <output_dir>:/train/output
+  -v <package-path>:/train/package
+  -e NUM_PASS=4 <paddle_image>
```

Comment:
I guess we also need `-e "PYTHONPATH=/train/package` so to enable Python finding imported packages there?